File: PRAK 402/PRAK402_2310817110011_Ghani_MUDZAKIR.py
nilai_awal = int(input())

for x in range (1, nilai_awal+1):
    if x%2 !=0:
        print(x, end= " ")
    else:
        continue

print("\n")
for z in range (nilai_awal, 0,-1):
    if z%2 ==0:
        print(z, end= " ")

    else:
        continue

File: PRAK 402/PRAK402_2310817110011_Ghani-Mudzakir.c
#include <stdio.h>

int main()
{
    int nilai_Awal;
    scanf("%d", &nilai_Awal);
    int i,j;
    for (i = 1; i <=nilai_Awal; i++)
    {
        if (i%2 !=0)
        {
            printf("%d ", i);
        }else{
            continue;
        
    }

    }
    printf("\n");
    for (j = nilai_Awal; j > 0; j--)
    {

        if (j%2 ==0)
        {
            printf("%d ", j);
        }

    }
    
    return 0;
}



